File: poo-noite/aula02/edu/curso/HelloWorld.java
package edu.curso;

public class HelloWorld { 

    int a = 10;

    public void fazAlgo() { 
        int b = 20;
        System.out.println("Faz algo esta sendo executada");
    }

    public static void main(String[] args) { 

        HelloWorld h1 = new HelloWorld();
        h1.a = 50;
        h1.fazAlgo();
        System.out.println("A: " + h1.a);

        System.out.println("Hello World");
    }
}

File: pdmi/aula01/HelloWorld/app/src/main/java/edu/curso/helloworld/HelloWorld.kt
package edu.curso.helloworld

import android.app.Activity
import android.os.Bundle
import android.widget.TextView

class HelloWorld : Activity() {
    override fun onCreate(bundle : Bundle?) {
        super.onCreate(bundle)

        val txt = TextView( this )
        // txt.setText("Hello world")
        txt.text = "Hello World"

        setContentView( txt )
    }
}